client/src/services/cryptpad.ts

```diff
@@ -0,0 +1,160 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+import { Env } from '@/services/environment';
+
+export enum CryptpadDocumentType {
+  Pad = 'pad',
+  Sheet = 'sheet',
+  Doc = 'doc',
+  Presentation = 'presentation',
+  Code = 'code',
+  Unsupported = 'unsupported',
+}
+
+export const ENABLED_DOCUMENT_TYPES = [
+  CryptpadDocumentType.Pad,
+  CryptpadDocumentType.Sheet,
+  CryptpadDocumentType.Doc,
+  CryptpadDocumentType.Presentation,
+  CryptpadDocumentType.Code,
+];
+
+export interface CryptpadConfig {
```

Comment:
This config is Cryptpad internal config, I'm not sure we should expose it directly, we can probably simplify the API.